toptal/granite
# Create official website documentation

We need to have some marketing about our framework and also collect results.
This PR sets the initial documentation website powered by [mkdocs](https://github.com/mkdocs/mkdocs/). I also created an analytics account for it to collect results and check the engagement.

## TODO

- [x] Deploy it
- [x] Add a link to the official website in the Github project
- [ ] Check if we have a granite logo or if we'll keep the Toptal logo

### Preview

https://toptal.github.io/granite/

### How to test

Install mkdocs and mkdocs-material:

```
pip install mkdocs
pip install mkdocs-material
```

Run the server:

```
mkdocs serve
```

It should be live on [127.0.0.1:8000/](http://127.0.0.1:8000/)

### How to deploy

```
mkdocs gh-deploy
```

### Review

- [x] Document code according to [Getting Started with Yard](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md).
- [ ] Get approval.

### Pre-merge checklist
